### PR TITLE
(RHEL-35703) manager: restrict Dump*() to privileged callers or ratelimit

### PR DESCRIPTION
**Pull request by lnykryn**

### CODE REVIEW

**Comment by lnykryn** [comment]
The failure from FuzzLite seems to be unrelated, but it should be fixed with https://github.com/redhat-plumbers/systemd-rhel9/pull/269

**Review by dtardon — APPROVED** [review]
LGTM

**Comment by jamacku** [comment]
@mrc0mmand Could you please have a look at CentOS CI? Thank you :pray:

File: test/units/testsuite-65.sh [hunk 1]
```diff
@@ -51,6 +51,21 @@ systemd-analyze dot --require systemd-journald.service systemd-logind.service >/
 systemd-analyze dot "systemd-*.service" >/dev/null
 (! systemd-analyze dot systemd-journald.service systemd-logind.service "*" bbb ccc)
 # dump
+# this should be rate limited to 10 calls in 10 minutes for unprivileged callers
```
Comment (mrc0mmand) [inline test/units/testsuite-65.sh:54]: We don't have the `test/units/util.sh` script in RHEL9, so you'll need to copy the definition of `runas()` here:

https://github.com/systemd/systemd/blob/e3828d7103a99a15a1e947ba3063294ead590631/test/units/util.sh#L133C1-L137C2

**Review by msekletar — APPROVED** [review]

File: test/units/testsuite-65.sh (hunk 1 above)
Comment (jamacku) [reply test/units/testsuite-65.sh:61]: addressed in https://github.com/redhat-plumbers/systemd-rhel9/pull/267/commits/27e836bd3eacb25d0a714cf59b2272316670ae26

**Pull request merged**